### PR TITLE
SYNR-1162

### PR DESCRIPTION
**Pull request by kimyen**

### CODE REVIEW

File: R/Utility.R [hunk 1]
```diff
@@ -123,3 +123,17 @@ guessPythonExePathEnvironmentVariables <- function(){
     pythonExePaths
 }
 
+getPandasDataFrame <- function(key){
+  pyExec(sprintf("x = %s.to_dict(orient='list')", key))
+  return( as.data.frame(pyGet("x"), optional=TRUE, stringsAsFactors=FALSE) )
+}
+
+getOrderedDict <- function(key){
```
Comment (brucehoff) [inline R/Utility.R]: need a test in which a null value is in the OrderedDict, but NOT at the end.  Is there a case in which a *key* is null?

File: tests/testthat/test_pyCall.R [hunk 2]
```diff
@@ -16,3 +16,16 @@ test_that("pyCall() returns r data.frame for pandas DataFrame", {
...
 })
+
+test_that("pyCall() returns named list for OrderedList", {
+  pyImport("OrderedDict", from="collections")
+  pyExec("od = OrderedDict([('pear', 1), ('apple', 4), ('orange', 2), ('banana', 3)])")
+  # define a test function in python
+  pyExec("return_named_list = lambda: od")
+  r_value <- pyCall("return_named_list")
+  expect_equal("numeric", class(r_value))
+  expect_equal("numeric", class(r_value))
```
Comment (brucehoff) [inline tests/testthat/test_pyCall.R]: names(r_value)
